### PR TITLE
Create 2022-04-30-Fei-Protocol.md

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

**Comment by kishoian** [comment]
Can I ask?

It seems to me that the categories are not quite right. Bridge Hack and Flash loan attacks are always DeFI and Protocol Exploit

If we use categories like 51% and Custodian it would be better to add only DeFi here. 
Bridge Hack, Flash loan, Phishing, etc - additional different tags

File: content/attacks/posts/2022-04-30-Fei-Protocol.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- FeiProtocol-Fuse Exploiter: [0x6162759eDAd730152F0dF8115c698a42E666157F](https://etherscan.io/address/0x6162759edad730152f0df8115c698a42e666157f)
+
+## Losses
+
+$80 Million
+
+## Timeline
+
+- **April 30, 2022, 09:01:35 AM +UTC:** [The hacker exploited a reentrancy vulnerability in lending protocol](https://etherscan.io/tx/0xadbe5cf9269a001d50990d0c29075b402bcc3a0b0f3258821881621b787b35c6)
+- **April 30, 2022, 09:34 AM +UTC:** [Smart contract analysis firm Block Sec tweets about it](https://twitter.com/BlockSecTeam/status/1520350965274386433)
```
Comment (JediFaust) [inline content/attacks/posts/2022-04-30-Fei-Protocol.md:25]: Unnecessary point

File: content/attacks/posts/2022-04-30-Fei-Protocol.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- **April 30, 2022, 10:23:58 AM +UTC:** [Funds have started to be laundered through Tornado Cash.](https://etherscan.com/tx/0x61ee3d6fdf29f84c36ad828608af38b516869631f494326ed10f82ef36ddf3f9)
+
+## Security Failure Causes
+
+- **Reentrancy vulnerabilities occur in a smart contract:** The attacker [exploited two functions within the contracts of Fei Protocol](https://www.halborn.com/blog/post/explained-the-fei-protocol-hack-april-2022)): exitMarket and borrow. The exitMarket function is responsible for ensuring that a deposit is not being utilized as collateral for any loan, after which it permits the withdrawal of the deposit. Meanwhile, the borrow function permits a user to secure a loan by using a deposited asset as collateral. However, this function does not adhere to the check-effects-interaction pattern, rendering it susceptible to exploitation.
```
Comment (JediFaust) [inline content/attacks/posts/2022-04-30-Fei-Protocol.md:30]: Leave the title of the cause as Reentrancy Vulnerability

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Comment by JediFaust** [comment]
> Can I ask?
> 
> It seems to me that the categories are not quite right. Bridge Hack and Flash loan attacks are always DeFI and Protocol Exploit
> 
> If we use categories like 51% and Custodian it would be better to add only DeFi here. Bridge Hack, Flash loan, Phishing, etc - additional different tags

Good point, we are working on it. If you have any regarding ideas, feel free to share, would appreciate it.

File: content/attacks/posts/2022-04-30-Fei-Protocol.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2022-04-30
+target-entities: 
+- Fei Protocol
+entity-types:
+- DeFi
+- Stablecoin
+attack-types:
+- Smart Contract Exploit
```
Comment (JediFaust) [inline content/attacks/posts/2022-04-30-Fei-Protocol.md:9]: Add Reentrancy Attack as an attack type

File: content/attacks/posts/2022-04-30-Fei-Protocol.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2022-04-30
+target-entities: 
+- Fei Protocol
```
Comment (JediFaust) [inline content/attacks/posts/2022-04-30-Fei-Protocol.md:4]: Add `Rari Capital` as an entity

File: content/attacks/posts/2022-04-30-Fei-Protocol.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+entity-types:
+- DeFi
```
Comment (JediFaust) [inline content/attacks/posts/2022-04-30-Fei-Protocol.md:6]: Add `Lending Platform` as an entity type

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2022-04-30-Fei-Protocol.md (hunk 1 above)
Comment (JediFaust) [reply content/attacks/posts/2022-04-30-Fei-Protocol.md:31]: Status?

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, address requested changes and merge main

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, add `Lending Platform` as an entity type and `Rari Capital` as a target entity. Also, Shorten the security failure cause as `Reentrancy Vulnerability`

**Review by JediFaust — APPROVED** [review]

**Pull request merged**